src/sentry/coreapi.py

```diff
@@ -235,7 +235,7 @@ def project_from_auth(self, auth):
         if not pk.roles.store:
             raise APIUnauthorized('Key does not allow event storage access')
 
-        return Project.objects.get_from_cache(id=pk.project_id)
```

Comment:
It'd probably make sense to change this method name to something more representative of the return type since it doesn't return a project any more. Can this just be `project_id_from_auth`?